arangod/IResearch/IResearchRocksDBLink.cpp

```diff
@@ -42,8 +42,8 @@
 namespace arangodb {
 namespace iresearch {
 
-IResearchRocksDBLink::IResearchRocksDBLink(IndexId iid, LogicalCollection& collection)
-    : RocksDBIndex(iid, collection, IResearchLinkHelper::emptyIndexSlice(),
+IResearchRocksDBLink::IResearchRocksDBLink(IndexId iid, LogicalCollection& collection, uint64_t objectId)
+    : RocksDBIndex(iid, collection, IResearchLinkHelper::emptyIndexSlice(objectId).slice(),
```

Comment:
it seems we pass a `VPackSlice` to a temporary `VPackBuilder` object which doesn't look safe

Reply:
No, it is safe as slice is never stored inside